test/examples/ieee14bus.jl

```diff
@@ -0,0 +1,69 @@
+    "branch3"=> PiModelLine(from= "bus2", to = "bus3",y=1.1350191923073958-1im*4.781863151757718, y_shunt_km=0.0438/2, y_shunt_mk=0.0438/2),
+    "branch4"=> PiModelLine(from= "bus2", to = "bus4",y=1.686033150614943-1im*5.115838325872083, y_shunt_km=0.0340/2, y_shunt_mk=0.0340/2),
+    "branch5"=> PiModelLine(from= "bus2", to = "bus5",y=1.7011396670944048-1im*5.193927397969713, y_shunt_km=0.0346/2, y_shunt_mk=0.0346/2),
+    "branch6"=> PiModelLine(from= "bus3", to = "bus4",y=1.9859757099255606-1im*5.0688169775939205, y_shunt_km=0.0128/2, y_shunt_mk=0.0128/2),
+    "branch7"=> StaticLine(from= "bus4", to = "bus5",Y=6.840980661495672-1im*21.578553981691588),
+    "branch8"=> Transformer(from= "bus4", to = "bus7", y=0.0-1im*4.781943381790359, t_ratio=0.978),
+    "branch9"=> Transformer(from= "bus4", to = "bus9", y=0.0-1im*1.7979790715236075, t_ratio=0.969),
+    "branch10"=> Transformer(from= "bus5", to = "bus6", y=0.0-1im*3.967939052456154, t_ratio=0.932),
+    "branch11"=> StaticLine(from= "bus6", to = "bus11",Y=1.9550285631772604-1im*4.0940743442404415),
+    "branch12"=> StaticLine(from= "bus6", to = "bus12",Y=1.525967440450974-1im*3.1759639650294003),
+    "branch13"=> StaticLine(from= "bus6", to = "bus13",Y=3.0989274038379877-1im*6.102755448193116),
+    "branch14"=> StaticLine(from= "bus7", to = "bus8",Y=0.0-1im*5.676979846721544),
+    "branch15"=> StaticLine(from= "bus7", to = "bus9",Y=0.0-1im*9.09008271975275),
+    "branch16"=> StaticLine(from= "bus9", to = "bus10",Y=3.902049552447428-1im*10.365394127060915),
+    "branch17"=> StaticLine(from= "bus9", to = "bus14",Y=1.4240054870199312-1im*3.0290504569306034),
+    "branch18"=> StaticLine(from= "bus10", to = "bus11",Y=1.8808847537003996-1im*4.402943749460521),
+    "branch19"=> StaticLine(from= "bus12", to = "bus13",Y=2.4890245868219187-1im*2.251974626172212),
+    "branch20"=> StaticLine(from= "bus13", to = "bus14",Y=1.1369941578063267-1im*2.314963475105352));
+
+
+pg_build = PowerGrid(buses, branches)
+pg_read = read_powergrid(joinpath(@__DIR__, "ieee14bus.json"), Json)
+
+@testset "IEEE 14-bus stucture tests" begin
+    @test pg_build.nodes == pg_read.nodes
+    @test pg_build.lines == pg_read.lines
+    @test pg_build.graph == pg_read.graph
+end
+
+operationpoint = find_operationpoint(pg_build)
+timespan= (0.0,5.)
+
+fault1 = ChangeInitialConditions(node="bus1", var=:ω, f=Inc(0.2))
+fault2 = LineFailure(line_name="branch2", tspan_fault=(1.,5.))
+fault3 = PowerPerturbation(node="bus5", fault_power=0.0, tspan_fault=(1.,5.), var=:P)
+
+@testset "IEEE 14-bus solution tests" begin
+    solution1 = simulate(fault1, pg_build, operationpoint, timespan)
+    @test solution1.dqsol.retcode == :Success
+
+    solution2 = simulate(fault2, pg_build, operationpoint, timespan)
+    @test solution2.dqsol.retcode == :Success
+
+    solution3 = simulate(fault3, pg_build, operationpoint, timespan)
+    @test solution3.dqsol.retcode == :Success
+end
```

Comment:
@antonplietzsch @luap-pik  I think it would also be nice to check if the solution is reproduced